pkg/release/release.go

```diff
@@ -201,6 +206,38 @@ func (r *Release) EnsureInstalled(ctx context.Context, name string, version Vers
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created release %#q", name))
 	}
 
+	if isOperator {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("ensuring operator pod exists for release %#q", name))
+
+		c := r.Condition().PodExists(ctx, r.namespace, fmt.Sprintf("app=%s", name))
+		err := r.waitForConditions(ctx, c)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("ensured operator pod exists for release %#q", name))
+	}
+
+	var operatorPodName string
+	var operatorPodNamespace string = r.namespace
```

Comment:
I think you might need to same fallback as `InstallOperator` to the GS namespace. I am not sure if the namespacing is right everywhere currently.

Reply:
Yes. I thought it's a good opportunity to test if it is right.